include/phasar/PhasarLLVM/Pointer/LLVMPointsToGraph.h

```diff
@@ -76,7 +76,7 @@ class LLVMPointsToGraph : public LLVMPointsToInfo {
     /// This may contain a call or invoke instruction.
     const llvm::Value *V = nullptr;
     EdgeProperties() = default;
-    EdgeProperties(const llvm::Value *V);
+    EdgeProperties(const llvm::Value *v);
```

Comment:
Why did the name change from upper to lower case? Should not uppercase be the correct llvm style?